common/lib/capa/capa/templates/choicegroup.html

```diff
@@ -45,7 +46,7 @@
                 % elif input_type != 'radio' and choice_id in value:
                 checked="true"
                 % endif
-                /> ${choice_label}
+                /> ${HTML(choice_label)}
```

Comment:
If `choice_label` really contains HTML, it would be best to name it `choice_label_html`.  I know you didn't write this code, so do this if it is relatively simple.